MvcApplicationForCV/Models/Enum.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace MvcApplicationForCV.Models
+{
+    public class Enum
+    {
+        public enum EducationStatuss
+        {
+            Compleated,
```

Comment:
enumiem labāk likt vērtības klāt.. 
aka Completed = 1,
Stoped = 2,
InProgress=3.
ja ieliksi pa vidu kko jaunu un to vērtību nebūs.. visi esošie dati nebūs vairs ticami. :) 


un Completed nevis Compleated. :)